docs/installation/DownloadAndInstall.md

```diff
@@ -8,7 +8,7 @@ To download ScubaGoggles:
 3. Extract the folder in the zip file.
 
 ## Installing Python Dependencies
-Minimum required Python version to run the tool is `3.7.16`.
+As of August 2024, minimum required Python version to run the tool is `3.10.x` to support newer version of dependent packages. To support newer version of packages, you may need to update to a newer Python version.

```

Comment:
A few thoughts here:
- I recommend using a version number instead of a date
- "to support newer version of dependent packages": no need to justify why the minimum is 3.10, I'd recommend cutting this
- The last sentence: I think this sentence doesn't add much. See below for a suggested refactor.
```suggestion
As of ScubaGoggles v0.2.0, the minimum required Python version to run the tool is `3.10.x`. While it's possible that ScubaGoggles may work with different versions of Python, 3.10 is the version we've tested and ensured works with the versions of the modules listed in ScubaGoggles' [dependencies](requirements.txt).

To support newer version of packages, you may need to update to a newer Python version.

```